gems/pending/openstack/openstack_configuration_parser.rb

```diff
@@ -13,7 +13,8 @@ def parse(file_content)
   SECTION_REGEXP                       = /^\s*\[(.+?)\]\s*/
   # Its attribute e.g. rpc_thread_pool_size=64, or starting with a #, which is just comment marking a default
   # value. Limited to snake_case words for attribute names and = as delimiter.
-  ATTRIBUTE_REGEXP                     = /^\s*[#\;]*\s*([\w]+)\s*[=]\s*(.*?)\s*$/
+  ATTRIBUTE_REGEXP                     = /^\s*([\w]+)\s*[=]\s*(.*?)\s*$/
+  COMMENTED_ATTRIBUTE_REGEXP           = /^\s*[#\;]*([\w]+)\s*[=]\s*(.*?)\s*$/
```

Comment:
Doesn't this match everything `ATTRIBUTE_REGEXP` does? It looks to be all we need.